src/HotChocolate/ApolloFederation/src/ApolloFederation/EntityTypeInterceptor.cs

```diff
@@ -30,14 +31,46 @@ internal class EntityTypeInterceptor : TypeInterceptor
                 definition);
         }
 
+        public override void OnTypesInitialized(
+            IReadOnlyCollection<ITypeDiscoveryContext> discoveryContexts)
+        {
+            if (!_entityTypes.Any())
```

Comment:
Any is good with dotnet core. With framework i think it is not that good. as we still support netstandard 2.0 at some places i think it makes sense to just use Count != 0

Reply:
Didn't know that, adapted it accordingly